src/rules/require-types.ts

```diff
@@ -0,0 +1,71 @@
+import {
+	type Closure,
+	type Function,
+	type Method,
+	type Parameter,
+	type Property,
+} from 'php-parser';
+
+import { createRule } from '../utils/create-rule';
+import { extractName } from '../utils/extract-names';
+
+export const requireTypes = createRule({
```

Comment:
add `MessageIds` and `Options` like others?
this is the convention from ts-eslint

Reply:
Done, Why needed?

Reply:
mostly for consistnecy, but maybe we can leave it, idk